src/main/kotlin/lotto/view/InputView.kt

```
@@ -14,8 +19,24 @@ class InputView {
         return input
     }
 
+    private fun getInputs(
+        message: String,
+        count: Int,
+    ): List<String>? {
+        println(message)
+        val inputs = mutableListOf<String>()
+        repeat(count) {
+            val input = readlnOrNull()?.trim()
+            if (input.isNullOrEmpty()) return null
```

여기서 null 을 반환하는 대신 예외를 던지면 실패를 더 빨리잡아서 해결을 빨리할 수 있을 것 같아요! fail fast.

---

앗 넵!!